setup.py

```diff
@@ -87,6 +87,17 @@ def package_tree(pkgroot):
     platforms='any',
     provides=['vispy'],
     install_requires=['numpy'],
+    extras_requires={
+        'ipython-static': ['ipython'],
+        'ipython-vnc': ['ipython>=2'],
+        'ipython-webgl': ['ipython>=2', 'tornado'],
+        'pyglet': ['pyglet>=1.2'],
+        'pyqt4': [],  # Why is this on PyPI, but without downloads?
+        'pyqt5': [],  # Ditto.
```

Comment:
Would it not be better to comment these out then? We might give the impression that `pyqt4` deps will be installed, but they will not.